python/instrumentation/openinference-instrumentation-bedrock/README.md

```diff
@@ -68,6 +71,40 @@ response_body = json.loads(response.get("body").read())
 print(response_body["completion"])
 ```
 
+Alternatively, all calls to `converse` are instrumented and can be viewed in the `phoenix` UI.
+
+```python
+session = boto3.session.Session()
+client = session.client("bedrock-runtime")
+
+message1 = {
+            "role": "user",
+            "content": [{"text": "Create a list of 3 pop songs."}]
+}
+message2 = {
+        "role": "user",
+        "content": [{"text": "Make sure the songs are by artists from the United Kingdom."}]
+}
+messages = []
+
+messages.append(message1)
+response = client.converse(
+    modelId="anthropic.claude-3-5-sonnet-20240620-v1:0",
```

Comment:
is here a simpler way to call this? Like an alias for these id's or something

Reply:
I don't believe so unfortunately

https://us-east-1.console.aws.amazon.com/bedrock/home?region=us-east-1#/providers?model=anthropic.claude-3-5-sonnet-20240620-v1:0